### DISCUSSION TITLE
[General] Airdrop Wallet Not Eligible

### DISCUSSION THREAD

**Discussion by hackspaceit** [zkSync-Community-Hub/zksync-developers#602] [General]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Hello, I want to ask why my wallet which has been interacting with ZKSync for a long time is not eligible for the airdrop?
My wallet address is **0xaa66fc4433dfd254f05a720a8c8a2f1f2fbdcb58**
ZkSync explore on OKX [https://www.oklink.com/zksync/address/0xaa66fc4433dfd254f05a720a8c8a2f1f2fbdcb58](url)
ZkSync explore on [https://explorer.zksync.io/address/0xaa66fc4433dfd254f05a720a8c8a2f1f2fbdcb58](url)

Hopefully I can get the answer here because I have contributed to various zksync projects since a year ago and the proof is in the explore scan.

**Comment by bxpana** [comment]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.
